packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -207,11 +423,97 @@ class FlutterError extends AssertionError {
   ///
   /// All sentences in the error should be correctly punctuated (i.e.,
   /// do end the error message with a period).
-  @override
-  String get message => super.message;
+  ///
+  /// This constructor defers to the [new FlutterError.fromParts] constructor.
```

Comment:
Consider dropping the `new` here and below?

Reply:
I think that is a DartDoc convention based on the 90+ uses of this pattern in package:flutter. Presumably that lets the link go to the constructor call instead of the class definition.

Reply:
Oh, right. Makes sense. Thanks!